pkg/config/api.go

```diff
@@ -0,0 +1,96 @@
+package config
+
+import (
+	"strings"
+
+	"github.com/supabase/cli/internal/utils/diff"
+	v1API "github.com/supabase/cli/pkg/api"
+)
+
+type (
+	RemoteApi struct {
+		Enabled         bool     `toml:"enabled"`
+		Schemas         []string `toml:"schemas"`
+		ExtraSearchPath []string `toml:"extra_search_path"`
+		MaxRows         uint     `toml:"max_rows"`
+	}
```

Comment:
**question:**

This is what I ended up with, this is public facing so it can be imported inside `branching` and we can use it's methods. And basically contain the fields that will be actually shared between remote and local config.